Swift/BRCrypto/common/BRBlockChainDB.swift

```diff
@@ -1565,10 +1565,15 @@ public class BlockChainDB {
         }
     }
 
+    private func apiGetNetworkName (_ name: String) -> String {
+        let name = name.lowercased()
```

Comment:
The C code lowecases the name, but doing it here too has no impact.

We could do the testnet/ropsten substitution at the C layer as well, but I can see the pros and cons of that. What do you think?

Reply:
Let's leave it here for now; we are expecting the whole JSON_RPC to disappear.  Admittedly, I didn't look/consider the C layer...